src/test/java/test/dependent/github1380/GitHub1380Sample4.java

```diff
@@ -0,0 +1,21 @@
+package test.dependent.github1380;
```

Comment:
@juherr - This test by itself doesn't look proper to me because of the scenario its testing. 
We are saying `testMethodA` has a priority of 1 but it depends on a method `testMethodB` which has a priority of 2. Don't you think that by itself is a confusing statement ? Why would someone do this ? (Apart from the fact that they want to do it for the sake of it). Can you please help me understand this part ?